src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/DSA.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+
+using Internal.Cryptography;
+
+namespace System.Security.Cryptography
+{
+    public abstract class DSA : AsymmetricAlgorithm
```

Comment:
We'd probably want to add the following public members:
```C#
byte[] SignData(Stream, HashAlgorithmName);
byte[] SignData(byte[] buffer, HashAlgorithmName);
byte[] SignData(byte[] buffer, int offset, int count, HashAlgorithmName);
bool VerifyData(byte[] data, byte[] signature, HashAlgorithmName);
bool VerifyData(Stream, byte[] signature, HashAlgorithmName);
```
and the protected drivers thereof:
```C#
protected abstract byte[] HashData(byte[] data, int offset, int count, HashAlgorithmName hashAlgorithm);
protected abstract byte[] HashData(Stream data, HashAlgorithmName hashAlgorithm);
```

But we may already be in a bind for adding those, since Desktop doesn't have them yet.

@nguerrera: Thoughts?